src/DynamoPackages/Package.cs

```diff
@@ -572,21 +575,66 @@ internal void SetAsLoaded()
             RaisePropertyChanged(nameof(LoadState));
         }
 
+        AssemblyLoadContext GetPackageALC()
+        {
+            if (LoadedAssemblies.Any())
+            {
+                if (LoadedAssemblies.All(x => AssemblyLoadContext.GetLoadContext(x.Assembly) != AssemblyLoadContext.Default))
+                {
+                    return AssemblyLoadContext.GetLoadContext(LoadedAssemblies[0].Assembly);
+                }
+            }
+            return AssemblyLoadContext.Default;
+        }
+
+
+
         internal void UninstallCore(CustomNodeManager customNodeManager, PackageLoader packageLoader, IPreferences prefs)
         {
             if (LoadedAssemblies.Any())
             {
+                bool unloadAnyPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("UnloadAnyIsolatedPackage", false) ?? false;
+                bool unloadThisPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("UnloadIsolatedPackage_" + this.Name, false) ?? false;
```

Comment:
IMO it's better to let the package author control this in their package manifest or via some API instead of guessing.

Reply:
Not sure about API. Isolation does not seem like something a package author should worry about (too technical).
Maybe a simple config would make it easier to use/grasp